python/ray/serve/tests/test_logs.py

```diff
@@ -1,113 +0,0 @@
-import time
```

Comment:
This file wasn't be run in CI due to a missing main clause and when I ran it, it was slow and failed.